webpack.config.js

```diff
@@ -0,0 +1,54 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+const config = {
+  entry: "./src/app/App.js",
+  output: {
+    path: path.resolve(__dirname, "dist"),
+    filename: "App.js",
+  },
+
+  mode: "development",
+  devServer: {
+    static: {
```

Comment:
I don't know how the new webpack solves this but we were talking about `hot reload` if you remember. Does it work with that configuration even thou, you don't have it here in devServer `hot: true`?

https://webpack.js.org/guides/hot-module-replacement/